server.py

```diff
@@ -0,0 +1,352 @@
+import flask
+from flask import Flask, json
+import waitress
+
+from datetime import datetime, timedelta
+
+from core import Controller
+from models import FoodCategory, FoodDetails, CustomerDetails, CustOrderSelection, CustOrderStatus
+
+
+not_found = {"message": "Not Found"}
+success = {"message": "Success"}
+failed = {"message": "Failed"}
+
+ 
+def create_app(fos):
+    """
+    Creates the server app.
+    """
+    app = Flask('Food Ordering System')
+
+    ### employee routes
+    @app.route('/employee/add_food_category', methods=['POST'])
+    def add_food_category():
+        """
+        >> curl -H "Content-Type: application/json" -X POST -d "{\"name\":\"drinks\"}" "localhost:8080/employee/add_food_category"
+        """
+        name = flask.request.json["name"]
+        add = fos.add_food_category(name)
+        return json.dumps("{}".format(success if add else failed))
```

Comment:
Return the added food category resource instead, in JSON.

Reply:
Returned JSON